front50-gcs/src/main/java/com/netflix/spinnaker/front50/model/BucketDAO.java

```diff
@@ -107,8 +109,21 @@ public boolean isHealthy() {
 
 
   public T findById(String id) throws NotFoundException {
-      return service.loadCurrentObject(buildObjectKey(id), daoTypeName,
-                                       serializedClass);
+      return new SimpleHystrixCommand<T>(
+          getClass().getSimpleName(),
+          getClass().getSimpleName() + "-findById",
+          ClosureHelper.toClosure(args -> {
+              return service.loadCurrentObject(buildObjectKey(id), daoTypeName,
```

Comment:
Bit of an odd line break but otherwise LGTM.

```
return service.loadCurrentObject(
  buildObjectKey(id, daoTypeName, serializedClass)
);
```

^^ would be a little more consistent I think.

Same below with the String.format().